docs/ref/settings.txt

```diff
@@ -1484,7 +1484,11 @@ This value mirrors the functionality and caveats of the
 ``FILE_UPLOAD_PERMISSIONS``
 ---------------------------
 
-Default: ``None``
+Default: ``0o644``
+
+.. versionchanged:: 3.0
```

Comment:
Hi @Himanshu-Lakhara. 

Just one more change here. From the Writing Documentation guide: 

> Put the changed annotation notes at the bottom of a section, not the top.

Can you just move this to the bottom of this section. 

Thanks!